src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Onramps style guide
+contentType: page
+template: basicDoc
+topics:
+  - onramps
+---
+
+import apmOnrampsIssuesExample from 'images/apm_screenshot-crop_onramps-issues-example.webp'
+
+import apmActiveEmptyState from 'images/apm_screenshot-crop_active-empty-state.webp'
+
+Onramps content is information that appears our UI when a user clicks the **?** icon. This content is meant to be truncated and contextualized to where the user is and what they need to know to perform a task.
+
+## How should I name the section titles?
```

Comment:
Think we should say that headers can be any format, as long as they're short/succinct. We have quite a few that are questions, and we have quite a few that are not questions, and maybe we specifically say there is no rule around that.

Reply:
Recommendation for heading: make sure it's clear what the heading relates to. For example, don't say, "What is this?" -- instead say "What's an issue?" This is important because the info is off the side of the page, and if you change the page the onramps stay open.